packages/ihe-gateway/server/CodeTemplates/Common Library/Read document from AWS S3 bucket/Read document from AWS S3 bucket.js

```diff
@@ -24,3 +24,25 @@ function xcaReadFromFile(path) {
 
   return result;
 }
+
+function xcaReadFromFileB64(path) {
```

Comment:
function that reads file as b64 for insertion into documentReference for inbound iti39 response

Reply:
Is this specific to XCA? If not, can we name it with a more generic name, like `getFileContentsAsB64FromS3()` or similar, like we have on Core's `s3.ts`, please?

Reply:
Yah it is specific for XCA. Thats the only place it makes sense to be used.

Reply:
Fine for now, but this is not XCA specific IMO. There's nothing that ties this logic to XCA, other than the fact its the only place that uses it - and with that thinking would would not have shared libraries.